src/gatling/simulations/scenarios/Barrister_PRL_C100.scala

```diff
@@ -0,0 +1,182 @@
+package scenarios
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import utils.{Common, Environment, Headers}
+
+/*===============================================================================================================
+* Court Admin C100 case progression. Send to local court --> Send to Gatekeeper --> Add an order --> Serve 
+================================================================================================================*/
+
+object Barrister_PRL_C100 {
+
+  val BaseURL = Environment.baseURL
+
+  val MinThinkTime = Environment.minThinkTime
+  val MaxThinkTime = Environment.maxThinkTime
+
+  exec(_.setAll(
+    "PRLRandomString" -> (Common.randomString(7)),
+    "PRLRandomPhone" -> (Common.randomNumber(8)),
+    "PRLAppDobDay" -> Common.getDay(),
+    "PRLAppDobMonth" -> Common.getMonth(),
+    "JudgeFirstName" -> (Common.randomString(4) + "judgefirst"),
+    "JudgeLastName" -> (Common.randomString(4) + "judgeLast"),
+    "todayDate" -> Common.getDate(),
+    "LegalAdviserName" -> (Common.randomString(4) + " " + Common.randomString(4) + "legAdv")))
+
+  val AddDocumentC100Applicant =
+
+    /*======================================================================================
+    * Run Manage Document task
+    ======================================================================================*/
+
+    group("XUI_PRL_C100progress_200_010_ApplicantAddDocument") {
```

Comment:
I think you've grouped this incorrectly, I think this request and the subsequent request with `ignore-warning=false`  should be grouped as one??